src/editor-plugin/plugin.js

```diff
@@ -68,9 +70,30 @@ const EditorPlugin = () => {
 				help={ __( 'This adjusts the visibility of a post in the fediverse, but note that it won\'t affect how the post appears on the blog.', 'activitypub' ) }
 				selected={ meta?.activitypub_content_visibility || 'public' }
 				options={ [
-					{ label: labelWithIcon( __( 'Public', 'activitypub' ), globe ), value: 'public' },
-					{ label: labelWithIcon( __( 'Quiet public', 'activitypub' ), people ), value: 'quiet_public' },
-					{ label: labelWithIcon( __( 'Do not federate', 'activitypub' ), notAllowed ), value: 'local' },
+					{
+						label: enhancedLabel(
+							__( 'Public', 'activitypub' ),
+							globe,
+							__( 'Post will be visible to everyone and appear in public timelines.', 'activitypub' )
```

Comment:
Should we swap the args order between icon and tooltip?

Reply:
I have no preference... if so, I would prefer: 1. icon 2. label 3. tooltip

Reply:
Fixed in [b89753a](https://github.com/Automattic/wordpress-activitypub/pull/1345/commits/b89753aeed989bd9cc68ee64b9b60a3f2b1a108c)